e2eAIOK/DeNas/scores/transformer_proxy.py

```diff
@@ -67,6 +71,15 @@ def get_linear_layer_metric_array(model_type, net, metric):
                 for sub_layer in layer.layers:
                     metric_array.append(metric(sub_layer.pos_ffn.fc1))
                     metric_array.append(metric(sub_layer.pos_ffn.fc2))
+        elif model_type == "hf":
```

Comment:
any chance we can make it more extensible to any future works? It looks we need to naming all the modifiable layers for any new model_type

Reply:
Added a center point dictionary to store the modifiable layers, any new type of layers can be stored in the dictionary.